strongbox-storage/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/services/impl/ArtifactSearchServiceImpl.java

```diff
@@ -60,9 +59,7 @@ public SearchResults search(SearchRequest searchRequest)
                 {
                     if (storage.containsRepository(repositoryId))
                     {
-                        final String contextId = storage.getId() + ":" +
-                                                 repositoryId + ":" +
-                                                 IndexTypeEnum.LOCAL.getType();
+                        final String contextId = storage.getId() + ":" + repositoryId + ":local";
```

Comment:
@sbespalov : I have the feeling this hasn't been merged properly. If you have a look on the left, the code from the master uses `IndexTypeEnum.LOCAL.getType();` instead of just `":local"`. Could you please have another look at the differences between the master and your branch for this class? Thanks! :)